### PR TITLE
Pud 786 Dossier Target date

### PR DESCRIPTION
**Pull request by NataliaOsk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,28 +68,50 @@ and integration tests including pact tests:
 After `npm install`, files will be created under both .git/hooks and .husky, that will automatically lint (and fix) any staged files in your commits, plus run a type check.
 
 ## Running the app
-The easiest way to run the app is to use docker compose to create the service and all dependencies. This starts the latest published docker container for hmpps-auth and redis, a fake manage-recalls-api (wiremock) and a local build of the manage-recalls-ui. 
+In easiest way to run ui and api is to use
```
Comment (jonnywyatt) [inline README.md:71]: The easiest way?

File: README.md [hunk 2]
```diff
@@ -68,28 +68,50 @@ and integration tests including pact tests:
...
 
-`docker compose up`
-
-OR use the following script to run in the background and ensure the fake-mange-recalls-api is up to date and all services start correctly:
+`./start-local-services.sh`
 
-`scripts/start-local.sh` 
+script in manage-recall-e2e project.
```
Comment (jonnywyatt) [inline README.md:75]: It's manage-recalls-e2e-tests
maybe link to it?
https://github.com/ministryofjustice/manage-recalls-e2e-tests

File: README.md [hunk 3]
```diff
@@ -68,28 +68,50 @@ and integration tests including pact tests:
...
 
 Either way check that this has succeeded e.g. via login locally (`http://localhost:3000/`)
 with `PPUD_USER` / `password123456`.  
 This user has the `MANAGE_RECALLS` role that allows access to the service.
 
-### Running the app for development
+### Running the app for the development
+For development run of the ui run
```
Comment (jonnywyatt) [inline README.md:82]: You don't need to run this in addition to start-local-services.sh as that starts -ui already
You only need to run npm run kill then npm run start:e2e if you want to restart the -ui

File: server/views/pages/dossierRecallInformation.njk [hunk 4]
```diff
@@ -15,13 +15,10 @@
...
             {% endif %}
-            <h1 class="govuk-heading-xl">
+            <h1 class="govuk-heading-l">
                 Create a dossier for {{ person.firstName }} {{ person.lastName }} recall
-
-                <div class="govuk-heading-l govuk-!-margin-top-8">
-                    Recall information
-                </div>
             </h1>
+            <div class="govuk-body-l">Dossier due on {{ recall.dossierTargetDate | dateTime }} </div>
```
Comment (jonnywyatt) [inline server/views/pages/dossierRecallInformation.njk:21]: Can you add to the create-dossier-spec.js integration test, to check for this label?

**Review by jonnywyatt — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (NataliaOsk) [reply README.md:82]: Yes, in my head it is for development, when we run e2e but after seeing the issue we need to restart it. Any suggestions for rewording it?

File: README.md (hunk 2 above)
Comment (NataliaOsk) [reply README.md:75]: Fair, will add

**Pull request merged**